src/averageAges.js

```diff
@@ -14,12 +14,32 @@
  *
  * @return {number}
  */
+function countSummOfAges(people) {
```

Comment:
Consider renaming `countSummOfAges` to `calculateSumOfAges` for better naming consistency. 🤔